x-pack/plugins/profiling/server/lib/setup/fleet_policies.ts

```diff
@@ -0,0 +1,193 @@
+      },
+      metrics: {
+        elasticsearch: {
+          hosts: ['https://1b6c02856ea642a6ac14499b01507233.us-east-2.aws.elastic-cloud.com:443'],
+          api_key: 'woq-IoMBRbbiEbPugtWW:_iBmc1PdSout7sf5FCkEpA',
+        },
+      },
+      keyvalue_retention: {
+        // 60 days
+        age: '1440h',
+        // 200 Gib
+        size_bytes: 200 * 1024 * 1024 * 1024,
+        execution_interval: '12h',
+      },
+    },
+  };
+
+  const {
+    id,
+    revision,
+    updated_at: updateAt,
+    updated_by: updateBy,
+    ...apmPolicyModified
+  } = apmPolicy;
+
+  apmPolicyModified.inputs = apmPolicy.inputs.map((input) => {
+    return input.type === 'apm'
+      ? merge({}, input, { config: { 'apm-server': { value: profilingApmConfig } } })
+      : input;
+  });
+
+  await packagePolicyClient.update(soClient, esClient, id, apmPolicyModified);
+}
+
+const CLOUD_AGENT_POLICY_ID = 'policy-elastic-agent-on-cloud';
+const COLLECTOR_PACKAGE_POLICY_NAME = 'elastic-universal-profiling-collector';
+const SYMBOLIZER_PACKAGE_POLICY_NAME = 'elastic-universal-profiling-symbolizer';
+
+export async function validateCollectorPackagePolicy({
+  soClient,
+  packagePolicyClient,
+}: ProfilingSetupOptions): Promise<Partial<SetupState>> {
+  const packagePolicies = await packagePolicyClient.list(soClient, {});
+  return {
+    collector_policy: {
+      installed: packagePolicies.items.some((pkg) => pkg.name === COLLECTOR_PACKAGE_POLICY_NAME),
+    },
+  };
+}
+
+export async function createCollectorPackagePolicy({
+  client,
+  soClient,
+  packagePolicyClient,
+}: ProfilingSetupOptions) {
+  const packagePolicy = {
+    policy_id: CLOUD_AGENT_POLICY_ID,
+    enabled: true,
+    package: {
+      name: 'profiler_collector',
+      title: 'Universal Profiling Collector',
+      version: '8.9.0-preview',
+    },
+    name: COLLECTOR_PACKAGE_POLICY_NAME,
+    namespace: 'default',
+    inputs: [
+      {
+        policy_template: 'universal_profiling_collector',
+        enabled: true,
+        streams: [],
+        type: 'pf-elastic-collector',
+      },
+    ],
+  };
+  const esClient = client.getEsClient();
+  await packagePolicyClient.create(soClient, esClient, packagePolicy, {
+    force: true,
+  });
+}
+
+export async function validateSymbolizerPackagePolicy({
+  soClient,
+  packagePolicyClient,
+}: ProfilingSetupOptions): Promise<Partial<SetupState>> {
+  const packagePolicies = await packagePolicyClient.list(soClient, {});
+  return {
+    symbolizer_policy: {
+      installed: packagePolicies.items.some((pkg) => pkg.name === SYMBOLIZER_PACKAGE_POLICY_NAME),
+    },
+  };
+}
+
+export async function createSymbolizerPackagePolicy({
+  client,
+  soClient,
+  packagePolicyClient,
+}: ProfilingSetupOptions) {
+  const packagePolicy = {
+    policy_id: CLOUD_AGENT_POLICY_ID,
+    enabled: true,
+    package: {
+      name: 'profiler_symbolizer',
+      title: 'Universal Profiling Symbolizer',
+      version: '8.8.0-preview',
```

Comment:
Should this version be the same as the one above?

Reply:
I believe the symbolizer package was created a release cycle before the collector package. The version timelines aren't in sync. I just noticed today that the symbolizer package was upgrade to `8.8.1-preview`.

Reply:
As mentioned above, we now find and install the latest version for both packages.